state/model.go

```diff
@@ -214,6 +213,10 @@ type ModelArgs struct {
 	CloudCredential names.CloudCredentialTag
 
 	// Config is the model config.
+	//
+	// Deprecated. ModelConfig is now handled by the model config domain.
+	// Has values necessary for creating a model, e.g. name and uuid,
+	// however will not be used to set model config in settingsC.
```

Comment:
TODO: Deprecating rather than removing ModelArgs.Config is a mistake. It won't be filled in during migration import now. We need to add the model name and UUID to ModelArgs instead.

Reply:
Updated this morning to still be passing through model config. This will naturally disappear as we move models over to dqlite so not worth investing time in it at the moment.